samples/validation/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+# COPYRIGHT NOTICE: (c) 2022 Irnas. All rights reserved.
+
+cmake_minimum_required(VERSION 3.20)
+
+# create compile_commands.json for clang
+set(CMAKE_EXPORT_COMPILE_COMMANDS on)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(user-settings-sample-validation)
+
+zephyr_compile_options(-fdiagnostics-color=always)
+
+zephyr_include_directories(src)
+target_sources(app PRIVATE src/main.c)
```

Clean up the cmakelists file:

```suggestion

cmake_minimum_required(VERSION 3.20)

find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})

project(user-settings-sample-validation)

zephyr_include_directories(src)
target_sources(app PRIVATE src/main.c)
```

Please check if other files need similar cleanup.